server/player/player.go

```diff
@@ -442,6 +442,39 @@ func (d *Data) SetMapPosID(pos byte) {
 	d.mapPos = pos
 }
 
+func (d Data) noChange() {
+	d.Send(packetInventoryNoChange())
+}
+
+// UseItem of Data
+func (d *Data) UseItem(slot int16, id int32) {
+	if d.hp < 1 {
+		d.noChange()
+		return
+	}
+
+	item, err := d.getItem(2, slot)
```

Comment:
You will need to do the same here as well I suspect.

Reply:
That shouldn't be necessary if the id structure does not stray. This function is solely for usable items which should always be in the use inventory.